sources/packages/api/src/services/student-account-applications/student-account-applications.service.ts

```diff
@@ -134,6 +134,7 @@ export class StudentAccountApplicationsService extends RecordDataModelService<St
       city: studentProfile.city,
       postalCode: studentProfile.postalCode,
       selectedCountry: studentProfile.selectedCountry,
+      sinConsent: true,
```

Comment:
Can we please add a comment like below to this hard-coded value?
```ts
// The Ministry approval will happen based on a student profile submitted form
// that contains the SIN consent. If the consent is not provided by the student
// the Ministry approval would never happen, hence we are considering at this
 // moment that the student provided already the SIN consent.
sinConsent: true,
```

Reply:
@dheepak-aot, yes we can. So far we are not using the dynamic data from the form submitted anywhere in the API. The dynamic data is saved only in the JSONB column to render the form to Ministry to approve it.
We can read it from DB, but I do not have a major concern to have it hardcoded here.
This would be the only place where I would say that it is maybe possible to have to be harcoded.
That said, I do not see a big deal if the Team decides to get the value from the DB.

Reply:
The comment was added but the same property was removed from the `StudentInfo` and the SIN consent is also hardcoded in the saving student method `student.sinConsent = true;`.
If it will be hardcoded there it does not need to be anywhere else.

Reply:
I get your point. My only concern is that we are not reading from source of truth and instead assigning it based on how application is built now. But I am ok if other devs think that hard-coding here is fine.